### DISCUSSION TITLE
sh1mmer asking for auth key

### DISCUSSION THREAD

**Discussion by pazza2234** [3kh0/ext-remover#443] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so, I got it to work (kinda) to boot, but when I wait, it gives me a qr code and says "enter authorization key" but when I go to the link it says that the url was not found on this server. what to I enter. Do I give the code, and what do I do. 
![image](https://user-images.githubusercontent.com/81707002/214821507-107d9280-aeb0-4414-93aa-2ef46b0fbfb0.png)

**Comment by Aka-but-nice** [comment, marked as answer]
@pazza2234 Try it again!

**Reply by pazza2234** [reply]
try what again, do what? witch part again? sorry If im being rude. also if you mean by rebooting and doing it agean I have tryed that, I will do it agen later though.

**Reply by ghost** [reply]
How do i get the sh1mmer download i cant find it

**Reply by pazza2234** [reply]
@ttvbeghhf I recomend opining a new descution to get more people to help. enyways, about 14 hours ago the readme of ext-remover was updated with a new tutorial. you nead to get your own shimes, I will let you know If I find a good place to find one, but the "bord" that your chromebook is on determins on what shime you nead. you can find your bord by going to chrome://version and next to "stable-channel" in the platform catigory is your board. you can build the shim yourself, or find them online. I will let you know if I find a good shim. but to answer your question, on the new readme, if you are on windows you nead wax4windows. this will take the shim you downloaded or made, and modify it to have sh1mmer. the link to wax4windows is https://dl.sh1mmer.me/build-tools.

**Reply by pazza2234** [reply]
@Aka-unblock may you please tell me what to try again, witch part. this will help me. Thank you.
